Dockerfile

```diff
@@ -56,22 +58,33 @@ RUN curl -L https://github.com/cubicdaiya/ngx_small_light/archive/v${NGX_SMALL_L
     cd /tmp/ngx_small_light-${NGX_SMALL_LIGHT_VERSION} && \
     ./setup
 
+# Prepare ngx_mruby
+RUN curl -L https://github.com/matsumotory/ngx_mruby/archive/v${NGX_MRUBY_VERSION}.tar.gz > /tmp/ngx_mruby-${NGX_MRUBY_VERSION}.tar.gz && \
+    cd /tmp && \
+    tar zxf ngx_mruby-${NGX_MRUBY_VERSION}.tar.gz && \
+    rm ngx_mruby-${NGX_MRUBY_VERSION}.tar.gz && \
+    cd /tmp/ngx_mruby-${NGX_MRUBY_VERSION} && \
+    ./configure --with-ngx-src-root=/tmp/nginx-${NGINX_VERSION} && \
+    make build_mruby && \
+    make generate_gems_config
```

Comment:
ref. https://github.com/matsumotory/ngx_mruby/wiki/Install#or-configure-with---add-modulengx_mruby_src-for-nginx-and-make-it